controllers/v1/entities.js

```diff
@@ -441,6 +441,66 @@ module.exports = class Entities extends Abstract {
       }
     });
   }
+
+    /**
+  * @api {get} /kendra/api/v1/entities/getUsersByEntityAndRole/:entityId?role=""
+  * @apiVersion 1.0.0
+  * @apiName Get users by entityId and role.
+  * @apiGroup Entities
+  * @apiHeader {String} X-authenticated-user-token Authenticity token
```

Comment:
@deepakr123 - Make it internal access token only.

Reply:
resolved